xbmc/filesystem/MusicDatabaseDirectory/DirectoryNodeBoxsetDiscs.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (C) 2005-2019 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "DirectoryNodeBoxsetDiscs.h"
```

Comment:
`CDirectoryNodeBoxsetDiscs` needs renaming  `CDirectoryNodeDiscs` throughout because it applies to navigation generally not just boxed stes